pkg/util/kubeconfig.go

```diff
@@ -344,3 +344,29 @@ func SetCurrentContext(kubeCfgPath, name string) error {
 	}
 	return nil
 }
+
+// DeleteKubeConfigContext deletes the specified machine's kubeconfig context
+func DeleteKubeConfigContext(kubeCfgPath, machineName string) error {
+	kcfg, err := ReadConfigOrNew(kubeCfgPath)
+	if err != nil {
+		return errors.Wrap(err, "Error getting kubeconfig status")
```

Comment:
Does it make sense to start enforcing errors not to be capitalized? We currently have both in the minikube codebase, but I think we could start using the same pattern everywhere. WDYT? @kairen @medyagh